Source/ARTAuthOptions.h

```diff
@@ -91,6 +91,7 @@ ART_ASSUME_NONNULL_BEGIN
 - (NSString *)description;
 
 - (ARTAuthOptions *)mergeWith:(ARTAuthOptions *)precedenceOptions;
```

Comment:
OK, I'll guess it will go away later as now `authorize` also replaces rather than merging.